<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenPeripherals</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>220</y>
      <width>181</width>
      <height>57</height>
     </rect>
    </property>
    <property name="title">
     <string>Brightness</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QSlider" name="set_brightness">
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="set_effect_color">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>240</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="set_active_color">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>240</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="set_all">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>200</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set All</string>
    </property>
   </widget>
   <widget class="QWidget" name="buttons" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>621</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>220</y>
      <width>181</width>
      <height>57</height>
     </rect>
    </property>
    <property name="title">
     <string>Speed</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QSlider" name="set_speed">
       <property name="maximum">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>240</y>
      <width>132</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="direction_main">
       <property name="text">
        <string>L -&gt; R</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="direction_alt">
       <property name="text">
        <string>L &lt;- R</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="set_effect">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>138</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Spectrum</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Wave</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Circle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rainbow Fade</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Breathing</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Static Color</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Key Fade</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Key Ripple</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Horizontal Lines</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Sparkle</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Random</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Vertical Spectrum</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Zig Zag</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Black Hole</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Worms</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bounce</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Static Pastel</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Rainbow Center</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Off</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Custom</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="set_rainbow">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>220</y>
      <width>77</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Rainbow</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>161</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
